components/radio/index.jsx

```diff
@@ -43,9 +48,17 @@ const propTypes = {
 	 */
 	id: PropTypes.string,
 	/**
-	 * The string or element that is shown as both the title and the label for this radio input.
+	 * **Text labels for internationalization**
+	 * This object is merged with the default props object on every render.
+	 * * `heading`: Heading for the visual picker variant
+	 * * `label`: Label for the _enabled_ state of the Toggle variant. Defaults to "Enabled".
+	 * * `toggleDisabled`: Label for the _disabled_ state of the Toggle variant. Defaults to "Disabled". Note that this uses SLDS language, and meaning, of "Enabled" and "Disabled"; referring to the state of whatever the checkbox is _toggling_, not whether the checkbox itself is enabled or disabled.
+	 * * `toggleEnabled`: Label for the _enabled_ state of the Toggle variant. Defaults to "Enabled".
```

Comment:
This looks like it was directly copied from checkbox prop comments.

Reply:
Yes, indeed! I forgot to remove the comments. I have made the changes,